tests/test_pex_builder.py

```diff
@@ -266,3 +267,24 @@ def verify(pb):
             pb3.build(target)
             pb4 = build_from_req_pex(td4, target)
         verify(pb4)
+
+
+def test_prex_builder_script_from_pex_path(tmpdir):
```

Comment:
```suggestion
def test_pex_builder_script_from_pex_path(tmpdir):
```

Reply:
Thanks - you know my answer there - I'll save some trees.